core/src/main/java/ai/timefold/solver/core/config/util/ConfigUtils.java

```diff
@@ -295,7 +304,8 @@ public static int ceilDivide(int dividend, int divisor) {
         return (dividend / divisor) + correction;
     }
 
-    public static int resolvePoolSize(String propertyName, String value, String... magicValues) {
+    // TODO what to make of magicValues?
+    public static int resolvePoolSize(String propertyName, @NonNull String value, String... magicValues) {
```

Comment:
@triceo `magicValues` only used for generating the message during exception handling in here - `Arrays.toString` does allow passing null, so `@Nullable` might be fine as an annotation for `magicValues`. Does that sound correct?